twostep.opam

```
@@ -28,8 +28,9 @@ available: [ arch != "arm32" & arch != "x86_32" ]
 depends: [
   "ocaml" {>= "4.08.0"}
   "base" {>= "v0.9.3"}
+  "digestif" {>= "1.0.0"}
   "hex" {>= "1.2.0"}
-  "mirage-crypto" {>= "0.11.0"}
+  "mirage-crypto" {>= "1.0.0"}
   "mirage-crypto-rng"
```

```suggestion
  "mirage-crypto-rng" {>= "1.2.0"}
```

you'll need that for the Mirage_crypto_rng_unix.use_default

---

Thanks 🙏